### PR TITLE
Update translations: merchant

### PR DESCRIPTION
**Pull request by translation-platform**
This PR modifies translations in the project. It will append multiple commits. Feel free to merge anytime.

## Before you deploy
- [ ] Check translations for malicious HTML.
- [ ] This PR is [safe to rollback](https://development.shopify.io/guides/shipping/safe_to_merge#Signs_your_PR_is_safe_to_rollback).

## Questions?
- Visit [#help-i18n-and-translation](https://shopify.slack.com/messages/C7TJQLVC7) for questions about the this pull request.

@Shopify/intl-languages

### CODE REVIEW

**Review by movermeyer — DISMISSED** [review]

**Review by movermeyer — APPROVED** [review]

File: locales/nb.schema.json [hunk 1]
```diff
@@ -827,6 +845,17 @@
         },
         "social_sharing": {
           "name": "Knapper for deling på sosiale medier"
+        },
+        "share": {
+          "name": "Del",
+          "settings": {
+            "featured_image_info": {
+              "content": "Hvis du inkluderer en kobling i innlegg på sosiale medier, vil sidens fremhevede bilde vises som forhåndsvisningsbilde. [Finn ut mer] (https://help.shopify.com/en/manual/online-store/images/showing-social-media-thumbnail-images)."
```
Comment (martinamarien) [inline locales/nb.schema.json:853]: This will create a broken link 
```suggestion
              "content": "Hvis du inkluderer en kobling i innlegg på sosiale medier, vil sidens fremhevede bilde vises som forhåndsvisningsbilde. [Finn ut mer](https://help.shopify.com/en/manual/online-store/images/showing-social-media-thumbnail-images)."
```

File: locales/nb.schema.json [hunk 2]
```diff
@@ -827,6 +845,17 @@
...
+            },
+            "title_info": {
+              "content": "En butikktittel og -beskrivelse inkluderes med forhåndsvisningsbildet. [Finn ut mer] (https://help.shopify.com/en/manual/promoting-marketing/seo/adding-keywords#set-a-title-and-description-for-your-online-store)."
```
Comment (martinamarien) [inline locales/nb.schema.json:856]: This will create a broken link 

```suggestion
              "content": "En butikktittel og -beskrivelse inkluderes med forhåndsvisningsbildet. [Finn ut mer](https://help.shopify.com/en/manual/promoting-marketing/seo/adding-keywords#set-a-title-and-description-for-your-online-store)."
```

File: locales/nb.schema.json [hunk 3]
```diff
@@ -842,6 +871,12 @@
         },
         "paragraph": {
           "content": "For å endre utdrag, må du redigere [blogginnleggene.] (https://help.shopify.com/en/manual/online-store/blogs/writing-blogs#display-an-excerpt-from-a-blog-post)"
```
Comment (martinamarien) [inline locales/nb.schema.json:873]:
```suggestion
          "content": "For å endre utdrag, må du redigere [blogginnleggene.](https://help.shopify.com/en/manual/online-store/blogs/writing-blogs#display-an-excerpt-from-a-blog-post)"
```
Not part of this PR, but also a broken link

File: locales/ja.schema.json [hunk 4]
```diff
@@ -835,6 +853,17 @@
         },
         "social_sharing": {
           "name": "SNS共有ボタン"
+        },
+        "share": {
+          "name": "共有",
+          "settings": {
+            "featured_image_info": {
+              "content": "SNSの投稿にリンクを含めると、そのページの記事のサムネイルがプレビュー画像として表示されます。[詳しくはこちら](https://help.shopify.com/ja/manual/online-store/images/showing-social-media-thumbnail-images)。"
```
Comment (martinamarien) [inline locales/ja.schema.json:861]: Some links are being localized (`ja`) and others aren't. Is this expected?

File: locales/pt-BR.schema.json [hunk 5]
```diff
@@ -842,6 +871,12 @@
         },
         "paragraph": {
           "content": "Para alterar resumos, edite os [posts do blog].(https://help.shopify.com/pt-BR/manual/online-store/blogs/writing-blogs#exiba-o-resumo-de-um-post)"
```
Comment (martinamarien) [inline locales/pt-BR.schema.json:873]: Not part of this PR, but this will generate a broken link. 
```suggestion
          "content": "Para alterar resumos, edite os [posts do blog](https://help.shopify.com/pt-BR/manual/online-store/blogs/writing-blogs#exiba-o-resumo-de-um-post)."
```

File: locales/vi.schema.json [hunk 6]
```diff
@@ -842,6 +871,12 @@
         },
         "paragraph": {
           "content": "Để thay đổi đoạn trích, hãy chỉnh sửa [bài viết blog].(https://help.shopify.com/en/manual/online-store/blogs/writing-blogs#display-an-excerpt-from-a-blog-post)"
```
Comment (martinamarien) [inline locales/vi.schema.json:873]: Same here. Not part of this PR but I want to leave a comment because this will generate a broken link 
```suggestion
          "content": "Để thay đổi đoạn trích, hãy chỉnh sửa [bài viết blog](https://help.shopify.com/en/manual/online-store/blogs/writing-blogs#display-an-excerpt-from-a-blog-post)."
```

**Review by martinamarien — CHANGES_REQUESTED** [review]
There are some links that are being translated that will generate broken links within the editor. We will need to have these updated before merging.

**Comment by movermeyer** [comment]
Closing and force-pulling

File: locales/nb.schema.json (hunk 2 above)
Comment (movermeyer) [reply locales/nb.schema.json]: Fixed by @rafal-nedzarek-loc in the Smartling backend

File: locales/nb.schema.json (hunk 3 above)
Comment (movermeyer) [reply locales/nb.schema.json]: Fixed by @rafal-nedzarek-loc in the Smartling backend

File: locales/pt-BR.schema.json (hunk 5 above)
Comment (movermeyer) [reply locales/pt-BR.schema.json:873]: I've fixed this in the Smartling backend

File: locales/vi.schema.json (hunk 6 above)
Comment (movermeyer) [reply locales/vi.schema.json:873]: Fixed by @rafal-nedzarek-loc in the Smartling backend
